main.py

```diff
@@ -229,19 +254,30 @@ def get_input(prompt, default):
 
 
 if __name__ == "__main__":
-    # user inputs
+    # delete tables
+    delete = get_input("Delete table(s) on OEP", "no")
+
+    if delete != "no":
+        delete_table_folder = get_input("Path to folder for csv tables to be deleted on OEP", "delete/")
+        delete_table_folder = pathlib.Path(delete_table_folder)
+        load_oep_credentials()
+        delete_tables(delete_table_folder)
+
+    # user input
+    upload_folder = get_input("Table folder", "data/")
     check_table = get_input(
         "Enter specific table name to check or hit enter for 'all'", "all"
     )
     nomenclature_path = get_input("Path to 'SEDOS_Modellstruktur.xlsx'", "data/")
-    upload_folder = get_input("Input folder", "data/")
 
     # nomenclature check
     upload_folder = pathlib.Path(upload_folder)
     nomenclature_path = pathlib.Path(nomenclature_path)
     check_nomenclature_table(check_table, upload_folder)
 
     # upload
-    load_credentials()
+    if delete == "no":
```

Comment:
now OEP credentials are loaded anyways.
I think you can remove this section and load OPE credentials above in any case...